app.mjs

```diff
@@ -68,11 +70,14 @@ const main = async function() {
 		handlers[key] = await (Handlers[val.handler](tezos, val.args));
 	}
 
-	const dispatch_command = function(command, batch) {
+	const dispatch_command = async function(command, batch) {
 		let handler = handlers[command.handler];
 		if (handler) {
 			let handling_function = handler[command.name];
 			if (handling_function) {
+				if (handling_function.constructor.name == 'AsyncFunction') {
+					return await handling_function(command.args, batch);
+				}
 				return handling_function(command.args, batch);
```

Comment:
This is a small point, but maybe worth us thinking about for a couple of minutes:

We could avoid the new `if` statement in the lines about this, and instead change this line to `return await handling_function(command.args, batch);`, which will work regardless of whether `handling_function` is an async function.

However, since `async` isn't a no-op, this wouldn't be identical to what you've done. What are your thoughts?